joomla/joomla-cms
# [5.1] Fix missing entries in help index

Pull Request for Issue #32419 and related to PR #43250.

### Summary of Changes

Add missing language strings for help index and generate toc.json.

### Testing Instructions

Open the help index in the backend and check that all help documents are displayed.

![grafik](https://github.com/joomla/joomla-cms/assets/64533137/66bd78ac-a93c-4ee0-bc54-eb3623e9bb84)

Compare with https://docs.joomla.org/Category:Help_screen_5.0

### Actual result BEFORE applying this Pull Request

Entries missing in index / TOC

### Expected result AFTER applying this Pull Request

All entries are present

### Link to documentations
Please select:
